content/docs/user-guide/components/reference/terrain/terrain-macro-material.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Terrain Macro Material Component
+linktitle: Terrain Macro Material
+description: 'Open 3D Engine (O3DE) Terrain Macro Material component reference.'
+weight: 100
+---
+
+The **Terrain Terrain Macro Material** component provides a method of defining a macro-level appearance of a region of terrain.
+
+## Usage ##
+
+You assign a color and a normal texture by dragging texture assets to the **Color Texture** or **Normal Texture** fields, or by clicking {{< icon "file-folder.svg" >}}.
+Once the textures are assigned, you can adjust the appearance by using the **Normal Flip X** and **Normal Flip Y** toggles, and adjust the strength of the normals by using the **Normal Factor** slider. 
+You can configure the dimensions of the region by adjusting the [Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape)
+
+## Provider ##
+
+[Terrain Gem](/docs/user-guide/gems/reference/environment/terrain)
+
+## Dependencies ##
+
+[Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape)
+
+## Properties ##
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Color Texture** | The image that will be rendered on the terrain. | Texture | None |
+| **Normal Texture** | The texture that will be used as the normal map. | Texture | None |
+| **Normal Flip X** | Set to true to flip the normals about X. | bool | false |
```

Comment:
```suggestion
| **Normal Flip X** | Set to true to flip the normals about X. | Boolean | `False` |
```
Suggestion to match formatting of other component pages.

Reply:
Done